features/cellular/framework/AT/AT_CellularStack.h

```diff
@@ -118,6 +119,7 @@ class AT_CellularStack : public NetworkStack, public AT_CellularBase {
         bool started; // socket has been opened on modem stack
         bool tx_ready; // socket is ready for sending on modem stack
         bool rx_avail; // socket has data for reading on modem stack
+        bool tls_socket;
```

Comment:
Where is this used

Reply:
I'll add similar description as other fields. This is internally used by AT_CellularStack to define if socket is offloaded TLSSocket.